cmd/cloud-controller-manager/app/options/options.go

```diff
@@ -36,7 +35,7 @@ import (
 
 // CloudControllerManagerOptions is the main context object for the controller manager.
 type CloudControllerManagerOptions struct {
-	Generic cmoptions.GenericControllerManagerOptions
+	GenericControllerManagerOptions *cmoptions.ControllerManagerOptions
```

Comment:
would keep the field short as `Generic`. The context here is clear.

Reply:
revert it